tests/operations.go

```diff
@@ -387,7 +387,7 @@ func (ops *Operations) VerifyCapacity(pvcName, capacity string) bool {
 		desiredCapacity, _ := resource.ParseQuantity(capacity)
 		return (desiredCapacity.Cmp(actualCapacity) == 0)
```

Comment:
By any chance is it having unit changes and wait for that to be equal?
What is minikube version in CSI Travis?